src/pages/dataTableUtilities/actions/cellActions.js

```diff
@@ -50,9 +50,10 @@ export const editSensorName = (newValue, rowKey, route) => (dispatch, getState)
   const { data, keyExtractor } = selectPageState(getState());
 
   const objectToEdit = data.find(row => keyExtractor(row) === rowKey);
+  const { id } = objectToEdit;
 
   if (objectToEdit) {
-    UIDatabase.write(() => UIDatabase.update('Sensor', { ...objectToEdit, name: newValue }));
+    UIDatabase.write(() => UIDatabase.update('Sensor', { id, name: newValue }));
```

Comment:
uhh this should actually be removed.. hm

Reply:
I think I will make another PR for the stuff to be removed.. cleaner/easier to review? Let me know what you want!